### PR TITLE
Update Tutotial/Get-Started

### PR DESCRIPTION
**Pull request by vankichi**
### Description:

I updated the GetStarted document.
In this PR, we provide the new way to Deloy Vald in the tutorial.
The new way is that deploying minimum construction of Vald.
This means that we will deploy only `vald-agent-ngt`, `vald-lb-gateway`, `vald-discoverer`, and `vald-manager-index`.
So, this tutorial does not require using any Database.

(This PR requires author review)

I will update the construction of the tutorial overall at the next task.

### Environment:

- Go Version: 1.16
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [x] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/tutorial/get-started.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ Fashion-mnist is used as an example of a dataset.
 - helm: v3 ~
 - libhdf5 (_only required for this tutorial._)
 
-Helm and hdf5 is required for this tutorial. If helm or hdf5 is not installed, please install [helm](https://helm.sh/docs/intro/install) and [hdf5](https://www.hdfgroup.org/).
+Helm is used for deploy Vald on your kubernetes and Hdf5 is used for running example code.<br>
```
Comment (kevindiu) [inline docs/tutorial/get-started.md:16]: Maybe it is better?
```suggestion
Helm is used to deploy Vald on your Kubernetes and Hdf5 is used to decode the sample data file to run the example.<br>
```

**Review by kevindiu — APPROVED** [review]
LGTM

**Review by hlts2 — DISMISSED** [review]
LGTM

**Pull request merged**